City-of-Helsinki/drupal-helfi-rekry
# Uhf 5285 job listing content type

# [UHF-5285](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5285)

Added content type for job listings + task area taxonomy. (Task areas can be retireved as a list from the Helbit API).

How to test:
- Run `make drush-cr drush-cim`.
- Check that `helfi_rekry_content` is enabled. 
- Check that `job_listing` content type and `task_area` taxonomy exist
- Check [pathauto patterns](https://helfi-rekry.docker.so/fi/admin/config/search/path/patterns) regarding job listing post type. 
- Go over fields of Job Listing and determine that it matches what's been specced. Take a look at translations - does anything seems out of place or wrong?
